src/app/pages/visitsCalendar/visitsCalendar.component.html

```diff
@@ -48,49 +50,76 @@ <h2>{{modalDate}}, {{modalTime}}</h2>
-            <input #textbox type="text" [(ngModel)]="data" required>
+        <div *ngIf='!currentSlotTaken' style="margin-bottom: 10px; margin-top: 10px">
+            <input #textbox class="form-control" type="text" [(ngModel)]="data" required>
         </div>
 
         <div *ngIf='!currentSlotTaken && !imAPatient'>
-            <select #select [(ngModel)]="priority">
+            <select #select [(ngModel)]="priority" class="form-control">
                 <option *ngFor="let item of priorities" [value]="item.id">{{item.name}}</option>
             </select>
         </div>
 
         <div *ngIf='currentSlotTaken'>
-            <a (click)="unEnroll()" class="btn btn-default">
+            <button (click)="unEnroll()" class="btn btn-danger" style="margin-top: 10px;">
                 UnEnroll!
-            </a>
+            </button>
         </div>
 
-        <h3>Patient's data</h3>
+        <button type="button" class="btn btn-primary" style="margin-top: 10px; margin-bottom: 10px;" (click)="exportToGoogle()">Export to Google Calendar</button>
+
         <!--<div class="dropdown">-->
         <!--<button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{{patientName}}-->
         <!--<span class="caret"></span></button>-->
         <div *ngIf='imAPatient && currentSlotTaken'>
             You are already enrolled for this appointment!
         </div>
-        <div *ngIf='imAPatient && !currentSlotTaken'>
-            <a (click)="enrollMe()" class="btn btn-default">Enroll!</a>
+        <div *ngIf='isInSinglePatientMode() && !currentSlotTaken'>
+            <button (click)="enrollMe()" class="btn btn-success btn-lg">Enroll!</button>
         </div>
-        <div *ngIf='!imAPatient && currentSlotTaken'>
-            <a (click)="showPatientDetails()" class="btn btn-default">
-                Taken by: {{patient.account.personalDetails.firstName}} {{patient.account.personalDetails.lastName}}</a>
+
+        <div *ngIf='imAReceptionist && currentSlotTaken'>
+            <h4 (click)="showPatientDetails()" >
+                Taken by: {{patient.firstName}} {{patient.lastName}}</h4>
+        </div>
+
+        <div *ngIf='!imAPatient && !imAReceptionist && currentSlotTaken'>
+            <h4 (click)="showPatientDetails()" >
+               Taken by: {{patient.account.personalDetails.firstName}} {{patient.account.personalDetails.lastName}}</h4>
         </div>
-        <div *ngIf='!imAPatient && !currentSlotTaken'>
-            <ul>
-                <li *ngFor="let patient of patients">
-                    <a (click)="setPatient(patient)">
-                        {{patient.firstName}} {{patient.lastName}}
-                    </a>
-                </li>
-
-            </ul>
+
+        <div *ngIf='!isInSinglePatientMode() && !currentSlotTaken'>
+            <h3 class="well-lg">Patients available for enrollment</h3>
+            <!--Ze specjalną dedykacją dla doktora Piotra Nawrockiego i jego młodego potomka-->
```

Comment:
xDDDDD

Reply:
To on właśnie prosił, żeby lista pacjentów w modalu wyglądała tak jak ta nie w modalu